core/include/kernel/ts_store.h

```diff
@@ -59,6 +59,9 @@ struct ts_store_ops {
  * TS storage is searched in order of priority, where lower values are
  * tried first.
  *
+ * Separate stores are created for TAs and SPs. The user of the stores
+ * knows if it needs to access the TA store or if it needs to access the SP one.
+ *
```

Comment:
Remove this part and add a comment specific to `TEE_TS_REGISTER_SP_STORE` instead.